templates/user.html

```diff
@@ -15,21 +15,25 @@
   <meta property="og:description" content="See all your Open Source contributions in one place."/>
   <!-- Open Graph End -->
 
+  {{if (ne .ResponseType "iframe")}}
   <!-- Twitter Card Start -->
```

Comment:
Increase indent for content between tags? I was going to, but that will change the ownership of those lines in git blame :-(
If youre strongly pro -  Ill do it